src/test/java/com/ironcorelabs/tenantsecurity/kms/v1/CryptoUtilsTest.java

```diff
@@ -119,6 +133,16 @@ public void streamingEncryptWithNormalDecrypt() throws Exception {
         assertEquals(decryptedBytes, plaintext);
     }
 
+    public void phpDocumentWithLargeTenantIdDecrypt() throws Exception {
```

Comment:
Although this might match a value in a test in the PHP SDK, it doesn't seem like it is a "PHP document".